docs/docs/routing.md

```diff
@@ -13,6 +17,40 @@ Gatsby makes it easy to programmatically control your pages. Pages can be create
 
 See the [Creating and Modifying Pages](/docs/creating-and-modifying-pages) for more detail.
 
+When Gatsby creates pages it automatically generates a path to access them. This path will differ depending on how the page was defined.
+
+### Pages defined in `src/pages`
+
+Each `.js` file inside `src/pages` will generate its own page in your Gatsby site. The path for those pages matches the file structure it's found in.
+
+For example, `contact.js` will be found at `yoursite.com/contact`. And `home.js` will be found at `yoursite.com/home`. This works at whatever level the file is created. If `home.js` is moved to a directory called `information`, located inside `src/pages`, the page will now be found at `yoursite.com/information/contact`.
```

Comment:
```suggestion
For example, `contact.js` will be found at `yoursite.com/contact`. And `home.js` will be found at `yoursite.com/home`. This works at whatever level the file is created. If `contact.js` is moved to a directory called `information`, located inside `src/pages`, the page will now be found at `yoursite.com/information/contact`.
```

I think this (or the last part: `youtsite.com/information/contact`) would need to be swapped so they match 🙂